### ISSUE TITLE
Link in Readme to example demo is broken.

### ISSUE THREAD

**Issue by Argzero** [tinybigideas/jGravity#9]
If the link in question no longer works in general, or there is no suitable replacement, please respond to this issue and ping me and I will offer a link to a website that uses this. Otherwise, please fix the broken link.

**Comment by tinybigideas** [comment]
@Argzero yes sorry the link no longer works. If anything, I'd probably look to remove the link entirely. Do you have a live link to shre?

**Comment by exurd** [comment]
@tinybigideas you can create a live link yourself by using Github Pages in the settings for this repository.

It would then be available at [tinybigideas.github.io/jGravity](tinybigideas.github.io/jGravity).
